### DISCUSSION TITLE
Sorc  LVL 37 cant kill Baal.

### DISCUSSION THREAD

**Discussion by heiligerhuegel** [isid0re/SoloLeveling#118] [General]
Hello, just installed the bot yesterday and after around 22 hrs the bot is stuck on Baal - normal. 

It farms gold in act 1 and 2 , kills Diablo and goes for Baal but dies (chickens out) after spending all gold on potions and merc revives. 

Then it repeats to farm gold on chests in act 1 & 2 + Diablo kill. 
 
One run start to finish (respawn to dying to baal) takes around 1h15 min ( 9 runs so far ). 

Baal fight looks scuffed - teleports around the room and it seems like all the meteors are missing).
Im using the Infotag - Leveling, havent changed any setting in the config. 

On the act 1 goldfarm bot makes around 200Gold/NetProfit. after spending most of the gold on manapotions and spending mana on meteors that wont hit any mobs. 

Thanks for the help.

**Comment by heiligerhuegel** [comment]
https://youtu.be/pw6ataRmcHg

Heres the Video of a run, Baal right in the beginning is chaos.
While "grinding" Gold, gold is just wasted. 

and it failed pre diablo

**Comment by ciompek** [comment]
I have a similar observation. My soso reached ~40 after 22 hours of running, while a hammerdin is ~75. What i think is the respec into fb/Meteor. I'll try to make the 'middle' respec into blizzard and see what are the results.

**Reply by ciompek** [reply]
Ok I found the reason - Soso is maxing blizzard but using meteor. The reason is in this file: \libs\config\Builds\Sorceress.Leveling.js 
I have changed the skill 56 (meteor) for 59 (blizzard). She is killing much faster and getting less chickens

**Reply by isid0re** [reply]
I'm revising the sorc builds and I'll try this out. Thanks for posting it

**Reply by heiligerhuegel** [reply]
Awesome, changed the script, and everything works perfectly now ! 

Thanks ! :)

**Reply by ciompek** [reply]
One additional fix that helps Leveling Soso against Cold Immune - Config.AttackSkill = [-1, 59, 47, 59, 47, 56, 55];

so her basic attacks are Blizzard / FB, if mob is cold immune - she will use Meteor instead of Blizz. Works pretty well

We can say i basically switched the attacks - preferred is cold not fire.

**Comment by isid0re** [comment]
Hi everyone, glad your liking the bot so far. A few things here.

 The builds are being revised so your suggestions are helpful. 

As for the infotag line. You should only list the finalbuild you will want. If you use "Leveling" in the infotag it will not change the build after the first respec but still use the second respec. Which would waste the respec.

The bot has 2 respecs: Start >> Leveling >> finalBuild 

And finally, the bot has a hard level stop of 5 levels over the level cap if it fails the levelcheck. To ensure it doesnt stay to long in a difficulty and will continue progressing. In this case the level changes are at 35 and 65 so the hard stop limits normal to 40 and nightmsre to 70.

**Reply by heiligerhuegel** [reply]
Thanks for the Help. 
I like your work, keep on going ! 
cheers !

**Comment by Alphax1** [comment]
Oef,, changing 56 to 59. This was the thing I was struggling with. Ive been running 2 days of fails on Nightmare @diablo lol cuz it used meteor and not blizz

**Reply by isid0re** [reply]
The recent updates have correct this problem.
